catalogue/webapp/components/WorkTabbedNav/WorkTabbedNav.tsx

```diff
@@ -0,0 +1,47 @@
+import { Work } from '@weco/common/model/catalogue';
+import { FunctionComponent } from 'react';
+import SubNavigation from '@weco/common/views/components/SubNavigation/SubNavigation';
+import { listView, eye } from '@weco/common/icons';
+import { toLink as itemLink } from '@weco/common/views/components/ItemLink/ItemLink';
+
+type Props = {
+  work: Work;
+  selected: string;
```

Comment:
Could we use a more specific type here e.g.

```suggestion
  selected:'catalogueDetails' | imageViewer';
```

to help us when we want to create instances of this?